internal/transactions/transactions.go

```diff
@@ -153,6 +153,13 @@ func (r *transactionResult) String() string {
 			Events: r.result.Events,
 		}
 
+		if r.result != nil && !command.ContainsFlag(r.include, "fee-events") {
+			// last 3 events are fee events
+			if e.Events != nil && len(e.Events) >= 3 {
```

Comment:
Can you put this 3 in a constant. It will be easier to find if we change the amount of events the fee deduction emits.

Reply:
replace magic number with constant.